src/Test/Utilities/Portable/Diagnostics/DiagnosticDescription.cs

```diff
@@ -378,8 +378,10 @@ public override string ToString()
                 sb.Append('"');
             }
 
-            sb.Append(", isSuppressed: ");
-            sb.Append(_isSuppressed ? "true" : "false");
+            if (_isSuppressed)
```

Comment:
was there something about the new diagnostic baselines that made this change necessary?

Reply:
"isSuppressed: " was added in [https://github.com/dotnet/roslyn/pull/47486](https://github.com/dotnet/roslyn/pull/47486). Unfortunately, that means diagnostic differences included "isSuppressed: " even in cases where the value was "false".

---
In reply to: [490591638](https://github.com/dotnet/roslyn/pull/47576#discussion_r490591638) [](ancestors = 490591638)